src/test/java/org/broadinstitute/hellbender/tools/copynumber/plotting/PlotDenoisedCopyRatiosIntegrationTest.java

```diff
@@ -0,0 +1,167 @@
+package org.broadinstitute.hellbender.tools.copynumber.plotting;
```

Comment:
I'm assuming that you have looked at the plots themselves to ensure that they are okay...

Reply:
Yup!

Reply:
Great... ignore this comment then.